features/editionable-worldwide-organisations.feature

```diff
@@ -130,6 +130,22 @@ Feature: Editionable worldwide organisations
     And I upload a file attachment with the title "Beard Length Statistics 2014" and the file "dft_statistical_data_set_sample.csv"
     Then The "Beard Length Statistics 2014" attachment should have uploaded successfully
 
+  Scenario: Viewing audit history for the worldwide organisation and associated pages
+    Given a published editionable worldwide organisation "Test Worldwide Organisation" with a "Personal information charter" page
+    When I create a new edition of the "Test Worldwide Organisation" worldwide organisation
+    And I visit the pages tab for the worldwide organisation
```

Comment:
Quite a lot of `And`, but we need to test a new page and and existing edited page